diag_manager/fms_diag_object.F90

```diff
@@ -654,7 +663,7 @@ subroutine fms_diag_send_complete(this, time_step)
   character(len=128) :: error_string
   type(fmsDiagIbounds_type) :: bounds
   integer, dimension(:), allocatable :: file_ids !< Array of file IDs for a field
-  logical, parameter :: DEBUG_SC = .true. !< turn on output for debugging
+  logical, parameter :: DEBUG_SC = .false. !< turn on output for debugging
```

Comment:
You can't turn this on if it's a parameter

Reply:
For the debug output they are just local parameters for if your working on the code so I figured they would be changed manually.

If we want to make them more accessible we could do a CPP macro or namelist option instead but I think this should be a parameter either way since it's value shouldn't change over the course of this routine.

Reply:
If it's a parameter, then it can't be a namelist option.  It would be good to make it a namelist option.  This can be a TODO